recordedit/form.controller.js

```diff
@@ -415,6 +426,7 @@
                 vm.recordEditModel.rows[rowIndex][column.name] = tuple.displayname.value;
             }, function noDataSelected() {
                 // do nothing
+                // TODO: need catchall here
```

Comment:
user closed modal and we don't want to do anything. Remove the rejectCallback() and no catch is needed either.